Dockerfile.cockpit

```diff
@@ -0,0 +1,9 @@
+FROM welder:latest
```

Comment:
`welder/fedora:latest`

Reply:
The welder/fedora image as base image will cause login page issue. The page will be kept in login page.

Reply:
There's no `welder:latest` image either. The format of these images is repo/image:tag. What is your intended base image  then ? Where does it come from?

Reply:
found the latest implementation. lgtm.